roles/confluent.control_center/tasks/main.yml

```diff
@@ -14,6 +14,46 @@
     - ansible_os_family
     - ansible_fqdn
 
+- name: Get Package Facts
+  package_facts:
+  when: installation_method == "package"
+
+- name: Stop Control Center Service if upgrade
```

Comment:
```suggestion
- name: Stop Control Center Service if Upgrading Packages
```

Simple task name change. Please change in other files. Sorry I missed this

Reply:
NP, I'll fix all locally following the suggested pattern.

Reply:
Fixed